service/src/main/resources/db/changesets/20231208_clone-workflows-app-resources.yaml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+- changeSet:
+    id: Set cloneInstructions to COPY_RESOURCE for WORKFLOWS_APP resources.
+    author: mspector
+    changes:
+    - update:
+        tableName: resource
+        columns:
+            - column:
+                name: cloning_instructions
+                value: "COPY_RESOURCE"
+        where:
+          or:
```

Comment:
Maybe I am missing something, but what is the purpose of this `or`? 🤔

Reply:
it's part of the "where" clause - it applies "or" to the two "and" conditions below.

@dvoet just suggested I write this in sql for clarity - this is probably why

Reply:
@salonishah11 , I'm sorry, my explanation was wrong - it's a typo, leftover from when I did implement it in the way I described